app/views/layouts/_sidebar.html.erb

```diff
@@ -146,8 +144,15 @@
             <%= sidebar_link_to "Controlling Concurrency", 'pipelines/controlling-concurrency' %>
             <%= sidebar_link_to "Branch Configuration", 'pipelines/branch-configuration' %>
             <%= sidebar_link_to "Scheduled Builds", 'pipelines/scheduled-builds' %>
-            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
+
+          </ul>
+          <p class="Docs__nav__section-subheading">Security</p>
+          <ul class="Docs__nav__sub-nav">
+            <%= sidebar_link_to "Secrets", 'pipelines/secrets' %>
             <%= sidebar_link_to "Audit Log", 'pipelines/audit-log' %>
+            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
+            <%= sidebar_link_to "Incoming Webhooks", 'pipelines/incoming-webhooks' %>
+            <%= sidebar_link_to "Removing Users", 'pipelines/removing-users' %>
```

Comment:
Do you think it makes sense to merge this into Users and Teams? I think it probably does, as long as we're clear which features are *Teams* and which are part of the default.

Reply:
At the moment, I'd actually like to keep them separate - that is, if I got you correctly and you mean that we could merge these 3 pages into 1?